assignment3/3_BasisFunKernel_STUDENT.md

```diff
@@ -312,9 +326,9 @@ plot_transform(np.linspace(-2,2,num=100), mu=[-1,0,1,0], s=[0.2, 0.2, 0.2, 1.],
 What do the mu and s parameters control for the sigmoid and the Gaussian basis functions?
 
 
-```python
 
-```
+mu: location of basis functions in input space
+s: governs spacial scale of basis functions
 
```

Comment:
Mu controls the mean so translation on the x axis, while s controls the steepness or spread (variance).